src/index.ts

```diff
@@ -10,12 +10,10 @@ export {WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT} from '.
 
 export {AppBridgeContext} from './utilities/app-bridge';
 
-/* eslint-disable @typescript-eslint/camelcase */
 export {
-  buildColors as UNSTABLE_buildColors,
-  roleVariants as UNSTABLE_roleVariants,
-  toCssCustomPropertySyntax as UNSTABLE_toCssCustomPropertySyntax,
-  Tokens as UNSTABLE_Tokens,
-  DefaultThemeColors as UNSTABLE_DefaultThemeColors,
+  buildColors,
```

Comment:
I thought very soon `buildColors`, `roleVariants` and `defaultThemeColors` will all come from polaris-tokens. I figured that'd mean they'd all be removed polaris-react entirely. With that in mind is there much point unprefixing them - all we'd be doing is making it so they can't be removed before v5?

I'm also pretty tempted to say that `toCssCustomPropertySyntax` should either be exposed from polaris-tokens, or not be exposed at all also as I'm not clear why it needs to be used externally.

I'm not sure why Tokens needs to exposed publicly?

Reply:
Good call. I'll finish up Tim's [PR incorporating this from Polaris-tokens](https://github.com/Shopify/polaris-react/pull/2682) first, rebase this PR, and then only export what I need for the documentation build script which imports from our compiled javascript.

Reply:
`Tokens` and `toCssCustomPropertySyntax` are both exposed for generating our Color System documentation. Since ultimately that will find a new home, I think it makes sense to keep them exported as `unstable` so folks don't use them and then find that they're gone (which would be a breaking change).

Reply:
Agree on keeping those tagged as unstable.

For the sake of :fire: we could duplicate `toCssCustomPropertySyntax` in  the color system doc builder then remove it from the exports as it's looks like a pretty trivial implementation